src/cli_agent_orchestrator/clients/tmux.py

```
@@ -317,13 +319,37 @@ def send_keys(
         """
+        # If paste-buffer is disabled, use send-keys instead (for user input)
+        if not use_paste_buffer:
+            logger.info(
+                f"send_keys (via send-keys): {session_name}:{window_name} - keys: {keys[:100]}..."
+            )
+            # Validate session and window names to prevent command injection
+            validated_session = validate_tmux_name(session_name, "session_name")
+            validated_window = validate_tmux_name(window_name, "window_name")
+            target = f"{validated_session}:{validated_window}"
+            # Send the text once, then send Enter separately enter_count times
+            subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)